{"status": "broken", "description": "Activating the pick action test and checking whether it was successful.\n        ", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "complex_test"}, {"name": "subSuite", "value": "TestComplexScenario"}, {"name": "host", "value": "Hyper-Dock"}, {"name": "thread", "value": "11466-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "tests.complex_test"}], "stop": 1608286887929, "uuid": "b4af38b5-f2d4-490d-b36b-24b3b31f351d", "historyId": "6df5cbf7c00ce5f14b4c42d69bcd2de9", "name": "test_verification_of_pick_action", "start": 1608286881207, "fullName": "tests.complex_test.TestComplexScenario#test_verification_of_pick_action", "statusDetails": {"message": "RuntimeError: No Trajectory execution capability available.", "trace": "self = <tests.complex_test.TestComplexScenario instance at 0x7fdff56ebd88>\n\n    def test_verification_of_pick_action(self):\n        \"\"\"Activating the pick action test and checking whether it was successful.\n        \"\"\"\n        mo = Model('glass')\n        data_logger('logger/logs/pick_action_start')\n        hx,hy,hz = mo.lucy_pos()[0],mo.lucy_pos()[1],mo.lucy_pos()[2]\n        # Pick from table and place on table\n        result = MoveItPickAndPlace( pick_x = 0, pick_y = 3.4 , pick_z = 0.54,\n>                                    place_x = 0.5, place_y = 3.5 , place_z = 0.52)\n\ntests/complex_test.py:174: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/action_client/pick_place_client.py:29: in __init__\n    base = moveit_commander.MoveGroupCommander(\"base\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <moveit_commander.move_group.MoveGroupCommander object at 0x7fe00b6d3210>\nname = 'base', robot_description = 'robot_description', ns = ''\n\n    def __init__(self, name, robot_description=\"robot_description\", ns=\"\"):\n        \"\"\" Specify the group name for which to construct this commander instance. Throws an exception if there is an initialization error. \"\"\"\n>       self._g = _moveit_move_group_interface.MoveGroupInterface(name, robot_description, ns)\nE       RuntimeError: No Trajectory execution capability available.\n\n/opt/ros/kinetic/lib/python2.7/dist-packages/moveit_commander/move_group.py:52: RuntimeError"}, "attachments": [{"source": "d613ed4e-b239-4b27-b802-461ffe7aa7b4-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "c7f76860-4f3d-4a88-a66b-3a57fa36a424-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "09bc0313-4873-466e-9ddb-ab6077a48b7d-attachment.txt", "type": "text/plain", "name": "stderr"}]}